agriculture/agriculturecore/views.py

```diff
@@ -252,10 +253,13 @@ def get_smart_farms(request):
         :class:`.JsonResponse`: A JSON response with the list of the Smart
             Farms within the DRM account.
     """
-    # Check if the AJAX request is valid.
```

Comment:
I think this file is not sync with master, these changes were done in one of the latest commits.